frontend/express/public/javascripts/countly/countly.template.js

```diff
@@ -1085,6 +1085,25 @@ var AppRouter = Backbone.Router.extend({
                     selectedCategory.find(".menu-category-title").addClass("active");
                 }
 
+                // hide empty section headers
+                var type = countlyGlobal.apps[store.get('countly_active_app')].type;
+                var categories = $('#' + type + '-type .menu-category');
+                for (var j = 0; j < categories.length; j++) {
+                    var children = categories[j].children;
+                    var isEmpty = true;
+                    for (var k = 0; k < children.length; k++) {
+                        if (children[k].className.indexOf('restrict') === -1 && children[k].className.indexOf('item') !== -1) {
+                            isEmpty = false;
+                        }
+                    }
+                    if (isEmpty) {
+                        $(categories[j]).hide();
+                    }
+                    else {
+                        $(categories[j]).show();
+                    }
+                }
+
```

Comment:
Can we do a different logic here?
Like by default all categories are hidden but as soon as at least one member is added here:
https://github.com/Countly/countly-server/blob/master/frontend/express/public/javascripts/countly/countly.template.js#L673

The reason for this is, because menu items can be added dynamically. For example, one of the plugins is waiting for data from API to check if menu item needs to be added or not. But submenu.init would already be called by that time.